NAMESPACE

```diff
@@ -42,11 +41,9 @@ export(timer_available_cancers)
 export(xCell.data)
 import(ComICS)
 import(dplyr)
-import(limSolve)
 import(mMCPcounter)
 import(magrittr)
 import(methods)
-import(preprocessCore)
```

Comment:
This is probably why the ABIS test failed, but should still be in here. Maybe need to Document again